src/FwdState.cc

```diff
@@ -906,7 +906,7 @@ FwdState::connectStart()
         }
 
         // Pinned connection failure.
-        debugs(17,2,HERE << "Pinned connection failed: " << pinned_connection);
+        debugs(17,2,"Pinned connection failed: " << pinned_connection);
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.